packages/dd-trace/test/config.spec.js

```diff
@@ -786,10 +786,23 @@ describe('Config', () => {
 
     expect(config.telemetry).to.not.be.undefined
     expect(config.telemetry.enabled).to.be.true
+    expect(config.telemetry.heartbeatInterval).to.be.eq(60000)
```

Comment:
`to.be.eq...` sounds weird IMO, the whole point of these asserting libs is that it sounds like normal english.
If you look at the rest of the repo:
without be 1000+ results
![image](https://user-images.githubusercontent.com/70517128/235648269-6d747c15-5485-4544-bafe-e41a69db2484.png)
with be 100+ results
![image](https://user-images.githubusercontent.com/70517128/235648334-b9978e18-a9fd-49e1-bfee-2cc00b302906.png)
There is a full order of magnitude difference, you even use `expect(options).to.deep.equal({` without the `.be` in a file below.
IMO the `eq`/`equal` keyword has the same grammar function as the `be` keyword.
Up to you to decide with the elements above.

Reply:
changed to `to.eq`